.travis.yml

```diff
@@ -75,69 +50,40 @@ jobs:
 
```

Comment:
Can you add 
```
- travis_wait make test-source-e2e
- travis_wait make test-json-format-output
```
Can be added here, so that we have three test file run per each cluster up

Reply:
not right now. It will break things, as tests are not cleaning up properly, and there is a mess with projects

Reply:
Got it. Thanks